commcare_connect/opportunity/forms.py

```diff
@@ -842,10 +897,10 @@ def __init__(self, *args, **kwargs):
                 </button>
                 """
             ),
-            Row(Field("payment_units")),
-            Field("max_total", wrapper_class="form-group col-md-4 mb-0"),
-            Field("max_daily", wrapper_class="form-group col-md-4 mb-0"),
-            Submit(name="submit", value="Submit"),
+            Row(
+                Submit(name="submit", value="Submit",css_class="button button-md primary-dark"),
```

Comment:
same thing about `float-end`

Reply:
`flex justify-end` is actually better than `float-end` and is supposed to work better in responsive layouts and interacts better with other elements than float-end.